rtmidi/midiutil.py

```diff
@@ -14,7 +14,7 @@
 
 __all__ = [
     'get_api_from_environment',
-    'list_avaiilable_ports',
```

Comment:
Ok, thanks! But if the changes unrelated to the topic of this PR get too many, I'd prefer having them in a separate PR (or just report them to me and I'll fix them).